docs/reference/commandline/node_ls.md

```diff
@@ -66,31 +68,55 @@ ID                           HOSTNAME        STATUS  AVAILABILITY  MANAGER STATU
 e216jshn25ckzbvmwlnh5jr3g *  swarm-manager1  Ready   Active        Leader
 ```
 
-### id
+#### ID
 
 The `id` filter matches all or part of a node's id.
 
 ```bash
 $ docker node ls -f id=1
-
```

Comment:
can you keep these blank lines? having an empty line makes it easier for people to distinguish the command that was run, and the output (even though it's not actually printed with a blank line)